include/net/nrf_cloud.h

```diff
@@ -984,6 +994,18 @@ bool nrf_cloud_fota_is_type_enabled(const enum nrf_cloud_fota_type type);
  */
 int nrf_cloud_fota_job_start(void);
 
+/**
+ * @brief Function to check if credentials exist in the configured location.
```

Comment:
```suggestion
 * @brief Check if credentials exist in the configured location.
```

Reply:
I have removed `Function to ...` from the other places in this file as well.